src/partitioner/local/Partitioner.cpp

```diff
@@ -0,0 +1,137 @@
+#include <algorithm>
```

Comment:
Lets rename this class to something like "FennelPartitioner". You have to put some comments explaining any deviations.

Reply:
Thanks @miyurud  for the feedback, Shall we keep this partitioner remain same and create separate classes for each streaming partitioning algorithms. This file would need to be refactored because currently, this file contains `Hash`, `LDG` and `Fennel` algorithms

WDYT ?

Reply:
+1, We can keep this Partitioner class if there is considerable common component across different partitioning algorithms. We should create separate Partitioner classes for each streaming partitioning algorithm. That will make things modular enabling easy understanding of the code segments.